apivoid/info.json

```diff
@@ -65,124 +68,78 @@
-                "continent_code": "",
-                "continent_name": ""
-              },
-              "category": {
-                "is_pastebin": "",
-                "is_anonymizer": "",
-                "is_code_sandbox": "",
-                "is_form_builder": "",
-                "is_free_hosting": "",
-                "is_url_shortener": "",
-                "is_free_dynamic_dns": "",
-                "is_free_file_sharing": ""
-              },
-              "blacklists": {
-                "engines": {
-                  "": {
-                    "engine": "",
-                    "elapsed": "",
-                    "detected": "",
-                    "reference": "",
-                    "confidence": ""
-                  }
-                },
-                "scantime": "",
-                "detections": "",
-                "engines_count": "",
-                "detection_rate": ""
-              },
-              "risk_score": {
-                "result": ""
-              },
-              "security_checks": {
-                "is_risky_category": "",
-                "is_most_abused_tld": "",
-                "website_popularity": "",
-                "is_domain_ipv4_valid": "",
-                "is_domain_blacklisted": "",
-                "is_domain_ipv4_private": "",
-                "is_domain_ipv4_assigned": "",
-                "is_domain_ipv4_loopback": "",
-                "is_domain_ipv4_reserved": "",
-                "is_uncommon_host_length": "",
-                "is_uncommon_dot_char_count": "",
-                "is_uncommon_dash_char_count": "",
-                "is_uncommon_clickable_domain": ""
-              }
-            }
-          },
-          "success": "",
-          "elapsed_time": "",
-          "credits_remained": "",
-          "estimated_queries": ""
+        "elapsed_ms": "",
+        "risk_score": {
+          "result": ""
+        },
+        "server_details": {
+          "ip": "",
+          "asn": "",
+          "isp": "",
+          "latitude": "",
+          "city_name": "",
+          "longitude": "",
+          "region_name": "",
+          "reverse_dns": "",
+          "country_code": "",
+          "country_name": "",
+          "continent_code": "",
+          "continent_name": ""
         },
-        "status": ""
+        "security_checks": {
+          "is_risky_category": "",
+          "is_most_abused_tld": "",
+          "website_popularity": "",
+          "is_domain_ipv4_valid": "",
+          "is_domain_blacklisted": "",
+          "is_domain_ipv4_private": "",
+          "is_domain_ipv4_assigned": "",
+          "is_domain_ipv4_loopback": "",
+          "is_domain_ipv4_reserved": "",
+          "is_uncommon_host_length": "",
+          "is_uncommon_dot_char_count": "",
+          "is_uncommon_dash_char_count": "",
+          "is_uncommon_clickable_domain": ""
+        }
       }
     },
     {
-      "operation": "iprep",
+      "operation": "get_ip_reputation",
```

Comment:
operation name change (in Info.json) is impacting customers' existing playbooks. In the customer's use case, the playbook resets the options (if we rename them)